src/main/java/Commands/UserCommand.java

```diff
@@ -0,0 +1,37 @@
+package Commands;
+
+import Exceptions.DukeException;
+
+/**
+ * The class Commands.UserCommand acts as a parent class for commands
+ */
+public class UserCommand implements Command {
```

Comment:
I'm curious as to why there is a distinction between `UserCommand` and `Command`. Do you foresee having other kinds of commands apart from `UserCommand` being used?

Reply:
yea maybe duke wants to make a command to force the user to do something
Just like carrot (bot)